plugins/auth-auth0/amplicationrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "settings": {
+    "AUTH0_DOMAIN": "AUTH0_DOMAIN",
+    "AUTH0_CLIENT_ID": "AUTH0_CLIENT_ID",
+    "AUTH0_AUDIENCE": "AUTH0_AUDIENCE",
+    "AUTH0_ISSUER_URL": "AUTH0_ISSUER_URL",
+    "recipe": {
```

Comment:
@Shurtu-gal let's try to be consistent with the naming convention at least within the plugin itself. I'd suggest to go for camelCase all the way

Reply:
Sure.